README.md

```diff
@@ -47,7 +47,22 @@ class Project extends Model implements Commentable
 }
 ```
 
-### Usage with Filament
+**4. Configure the Commentions paths in your Tailwind config:**
+
+If you don't have a [custom theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme), you will have to configure. This is Filament's [recommendation](https://filamentphp.com/docs/3.x/support/assets#using-tailwind-css-in-plugins) if you are using Plugins.
```

Comment:
> If you don't have a [custom theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme), you will have to add the following to your application's `tailwind.config.js` file. This is Filament's [recommendation](https://filamentphp.com/docs/3.x/support/assets#using-tailwind-css-in-plugins) if you are using Plugins.

Reply:
This message isn't accurate if I am reading the linked docs correctly. If users are building a custom theme then they need to have our files in the content being scanned by their Tailwind setup otherwise the classes from the plugin may not be included.

The warning is for users who rely on the default stylesheet because they won't be using this config file to build all their CSS classes.

Reply:
This is part of this full section which reads like this:

**4. Configure the Commentions paths in your Tailwind config:**

If you don't have a [custom theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme), you will have to add the following to your application's tailwind.config.js file. This is Filament's [recommendation](https://filamentphp.com/docs/3.x/support/assets#using-tailwind-css-in-plugins) if you are using Plugins.

```js
export default {
    content: [
        '...',
        './vendor/kirschbaum-development/commentions/resources/**/*.js',
        './vendor/kirschbaum-development/commentions/resources/views/**/*.blade.php',
    ],
    // ...
}
```

Reply:
Both approaches seem funky and not ideal to me tbh, but it is what it is

Reply:
I tested it out in a new Laravel project quick and on the panel builder without a custom theme, it seems to look fine in an infolist.

I am having some trouble testing it out with a custom theme because new laravel uses Tailwind 4 but Filament 3 still requires Tailwind 3 so creating the theme in a new project is a dependency mess, but if you don't have a custom theme, there was no need that I could see to add these lines to a tailwind config file, it worked with the pre-built stylesheet quite well.